github3/orgs.py

```diff
@@ -508,6 +508,20 @@ def public_events(self, number=-1, etag=None):
         url = self._build_url('events', base_url=self._api)
         return self._iter(int(number), url, Event, etag=etag)
 
+    @requires_auth
+    def invitations(self, number=-1, etag=None):
+        r"""Iterate over outstanding invitations to this organization.
+
+        Implements https://developer.github.com/v3/orgs/members/#list-pending-organization-invitations
+
+        :returns: generator of :class:dict
+        """
+        headers = {'Accept': 'application/vnd.github.korra-preview', }
+        params = {}
+        url = self._build_url('invitations', base_url=self._api)
+        return self._iter(int(number), url, dict, params=params, etag=etag,
```

Comment:
I considered an object, but didn't see the benefit at present:
- there are no api's to manipulate an invitation
- the payload is still changing

Particularly since the payload is still evolving ('inviter' is a recent addtion), many early adopters would need to use as_dict anyway. I think promoting it to an object before it is stable is a good time to apply the YAGNI principle.

Would it make sense to issue a deprecation warning on the current method? Switching to returning objects would break backwards compatibility.